core/templates/pages/exploration-player-page/services/stats-reporting.service.spec.ts

```diff
@@ -94,14 +94,6 @@ describe('Stats reporting service ', () => {
 
   it('should set session properties when calling ' +
     '\'initSession\'', fakeAsync(() => {
-    // Prechecks.
-    expect(statsReportingService.explorationId).toEqual(null);
-    expect(statsReportingService.explorationTitle).toEqual(null);
-    expect(statsReportingService.explorationVersion)
-      .toEqual(null);
-    expect(statsReportingService.sessionId).toEqual(null);
-    expect(statsReportingService.optionalCollectionId).toEqual(null);
```

Comment:
Here and below can we have relevant pre-checks like we did before?

Reply:
Done